src/main/java/com/massivecraft/factions/config/file/MainConfig.java

```diff
@@ -760,6 +760,35 @@ public boolean isMessageByChunk() {
             }
         }
 
+        public class Commands {
+
+            public class ShowClaims {
+                private List<String> show = new ArrayList<String>() {
+                    {
+                        this.add("{header}");
+                        this.add("&2&lOverworld {showclaims_world}");
```

Comment:
Doing it this way makes any sort of custom worlds support much trickier. Why not just a repeatable `{worldname} {claims}` pattern?

Reply:
I wouldn't know how that would work. 
Besides this should work for custom worlds just add a custom worldname after _showclaims_.
We can't get a fancy name for a worldname, it would have to be something like _world_0_ or _world_nether_

Reply:
I wouldn't know how that would work.
Besides this should work for custom worlds just add a custom worldname after showclaims.
We can't get a fancy name for a worldname, it would have to be something like **world_0** or **world_nether**

Reply:
Could create a section that maps world names to fancy names, with defaults.

```
worldNamesMap:
    world: overworld
    world_nether: nether
    world_end: end
```